packages/core/src/__tests__/dispatcher.test.ts

```diff
@@ -62,8 +69,16 @@ describe('Dispatcher', () => {
   })
 
   it('makes registration correctly', async () => {
-    const doc = new Document(DocID.fromString(FAKE_DOC_ID), dispatcher, null)
-    doc._doctype = new TileDoctypeMock()
+    const doc = new Document(
```

Comment:
How did this PR affect this?  Why did this have to change?

Reply:
Throws a bunch of console errors